consensus/cometbft/service/validate_execution_header.go

```
@@ -0,0 +1,144 @@
+		))
+	// emptyRoot := common.Root(
+	//	common.NewExecutionHashFromHex(
+	//		"0x0000000000000000000000000000000000000000000000000000000000000000",
+	//	))
+
+	// For genesis block (when block number is 0), ParentHash must be zero
+	if !bytes.Equal(header.ParentHash[:], zeroHash[:]) {
+		return errors.New("parent hash must be zero for genesis block")
+	}
+
+	//if header.StateRoot != emptyTrieRoot {
```

golangci-lint (1.62.2)

commentFormatting: put a space between `//` and comment text

(gocritic)